web-console/4.0-designs/help-menu/help-menu.md

```diff
@@ -0,0 +1,29 @@
+# Help Menu
+
+The help menu will be accessible at all times from the primary masthead. Users can access it by clicking the help menu dropdown (fa-question-circle-o).
+
+**Menu Dropdown**
+![menu](img/menu.png)
+
+* The menu dropdown will include links to Documentation, the Quick Start Guide, the Command Line Tools page, and the About modal
+  * Documentation will redirect users to an external link
+  * Quick Start Guide will take users to the Quick Start Guide landing page
+  * Command Line Tools will redirect users to the CLI page
+  * About will open the About modal
+
+
+
+**Quick Start Guide**
+![Quick Start Guide](img/quick-start-guide.png)
```

Comment:
@serenamarie125 It's not actually bold but it looks heavier because the high fidelity mock of the QSG panel used open sans light 13. I can change it and also update the quick start guide documentation?